oem.json

```diff
@@ -3139,7 +3294,7 @@
     },
     {
         "id": "com.motorola.comcastext",
-        "description": "See above. Provides special (useless) features from Comcast? App title is \"Activation\". Safe to remove (tested only on non-Comcast phone).\n",
+        "description": "See above. Provides special (useless) features from Comcast? App title is \"Activation\". Safe to remove (tested only on non-Comcast phone).",
```

Comment:
I think we should copy whatever is present in the “See above” section.

Reply:
Yes I noticed this _useless_ description. Unfortunately the package isn't available on my device hence can't comment.